packages/editor-ui/src/components/NodeSettings.vue

```diff
@@ -535,6 +640,21 @@ export default mixins(
 		},
 		mounted () {
 			this.setNodeValues();
+
+			if (!this.isHttpRequestNodeV2(this.node)) {
+				const activeNodeType = this.$store.getters.nodeType(this.node.type, this.node.typeVersion);
+
+				if (activeNodeType && activeNodeType.credentials) {
+					const credentialTypeNames = activeNodeType.credentials.map((c: { name: string }) => c.name);
+					this.checkHttpRequestNodeSupport(credentialTypeNames);
+				}
+			} else if (
+				this.isHttpRequestNodeV2(this.node) &&
+				this.node.parameters.authentication === 'existingCredentialType'
+			) {
+				this.prepareScopesNotice(this.node.parameters.nodeCredentialType as string);
```

Comment:
could not we have done this as a node property in HTTP node?

Reply:
Do you mean moving `prepareScopesNotice` as a `notice` param to HTTPRN? This needs access to the store and to `$locale` to create the notice. Sorry, not sure how you mean?

Reply:
ya that's what I meant. not sure what would have been required but just wondering